hazelcast/src/test/java/com/hazelcast/internal/config/override/ExternalMemberConfigurationOverrideEnvTest.java

```diff
@@ -90,14 +91,14 @@ public void shouldHandleAdvancedNetworkEndpointConfiguration() throws Exception
         ServerSocketEndpointConfig restEndpointConfig = (ServerSocketEndpointConfig) config.getAdvancedNetworkConfig().getEndpointConfigs().get(EndpointQualifier.REST);
         ServerSocketEndpointConfig memcacheEndpointConfig = (ServerSocketEndpointConfig) config.getAdvancedNetworkConfig().getEndpointConfigs().get(EndpointQualifier.MEMCACHE);
 
-        assertEquals(clientEndpointConfig.getPort(), 9000);
-        assertEquals(clientEndpointConfig.getPublicAddress(), "127.0.0.1");
-        assertEquals(memberEndpointConfig.getPort(), 9001);
-        assertEquals(memberEndpointConfig.getPublicAddress(), "127.0.0.2");
-        assertEquals(restEndpointConfig.getPort(), 9002);
-        assertEquals(restEndpointConfig.getPublicAddress(), "127.0.0.3");
-        assertEquals(memcacheEndpointConfig.getPort(), 9003);
-        assertEquals(memcacheEndpointConfig.getPublicAddress(), "127.0.0.4");
+        assertEquals(9000, clientEndpointConfig.getPort());
+        assertEquals("127.0.0.1", clientEndpointConfig.getPublicAddress());
+        assertEquals(9001, memberEndpointConfig.getPort());
+        assertEquals("127.0.0.2", memberEndpointConfig.getPublicAddress());
+        assertEquals(9002, restEndpointConfig.getPort());
+        assertEquals("127.0.0.3", restEndpointConfig.getPublicAddress());
+        assertEquals(9003, memcacheEndpointConfig.getPort());
+        assertEquals("127.0.0.4", memcacheEndpointConfig.getPublicAddress());
```

Comment:
I guess these changes belong to https://github.com/hazelcast/hazelcast/pull/17878 and have already been merged. If that's right, could you please rebase on master to review? Otherwise LGTM.

Reply:
I just messed up the order and put _expected_ in place of _actual_ in assertions. This change just reorders the parameters

Reply:
Ah, I see. Thumbs up then, thanks :+1: